.gitignore

```diff
@@ -1,2 +1,4 @@
 .vscode
-.env
\ No newline at end of file
+.env
+
+run.bat
```

Comment:
A `run.bat` file isn't used in the repo, so would mind reverting this change, please?